src/LightDFG/FactorGraphs/FactorGraphs.jl

```diff
@@ -31,19 +31,22 @@ export
     filter_vertices,
     reverse
 
-import DistributedFactorGraphs: DFGNode
-const AbstractNodeType = DFGNode
+# import DistributedFactorGraphs: DFGNode
+# const AbstractNodeType = DFGNode
+import DistributedFactorGraphs: AbstractDFGVariable, AbstractDFGFactor
+const AbstractVariableType = AbstractDFGVariable
+const AbstractFactorType = AbstractDFGFactor
```

Comment:
This is the way I wanted to do it from the start. So I'm more than happy with the new Abstract types

Reply:
Cool!